testing/lint_interactive.sh

```diff
@@ -1,19 +1,27 @@
-echo '==> STARTING ESLINT'
+#!/bin/bash
+
+reset="\e[0m"
+bold="\e[1m"
+green="\e[92m"
+yellow="\e[93m"
+red="\e[91m"
+
+echo -e "$bold$yellow==> STARTING ESLINT$reset"
 npm run lint
 if [ $? -eq 1 ]; then
-    echo '==> ESLINT FAILED'
-    echo 'If possible, do you want to auto-fix the issues? (y/N)'
+    echo -e "$bold$red==> ESLINT FAILED$reset"
+    echo -e "$boldIf possible, do you want to auto-fix the issues? (y/N)$reset"
     read userInput
     if [ "$userInput" = "y" ]; then
-        echo '==> Running eslint --fix'
+        echo -e "$bold$yellow==> Running eslint --fix$reset"
         npm run lint-fix
-        echo 're-running test...'
+        echo -e "$bold$yellow==> re-running test...$reset"
         npm test
     else
-        echo 'aborting autofix, returning error.'
+        echo -e "$bold$red==> aborting autofix, returning error.$reset"
         exit 1
     fi
 else
-echo '==> LINT COMPLETED WITHOUT ERRORS'
+echo -e "$bold$green==> LINT COMPLETED WITHOUT ERRORS$reset"
 exit 0
```

Comment:
Indent

Reply:
fixed in commit [15a155e](https://github.com/vattenbagarna/Frontend/pull/9/commits/15a155ef54e4e93e7d1a154de299a232ea8da33b)